movethebytes/data-transfer/rust/src/bin/krakensync.rs

```diff
@@ -0,0 +1,173 @@
+use std::borrow::Cow;
+use std::{
+    net::{IpAddr, SocketAddr},
+    time::Instant,
+};
+
+use anyhow::{anyhow, Result};
+use cid::Cid;
+use env_logger::Env;
+use futures::StreamExt;
+use log::info;
+
+use krakensync::{core::Store, network::Node};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    run_test().await?;
+
+    Ok(())
+}
+
+const LISTENING_PORT: u16 = 31131;
+
+async fn run_test() -> Result<()> {
+    info!("Running test!");
+
+    env_logger::Builder::from_env(Env::default().default_filter_or("info")).init();
+    let client = testground::client::Client::new_and_init()
+        .await
+        .map_err(|e| anyhow!("{:?}", e))?;
+
+    let seq_num = client.global_seq();
+
+    match seq_num % 2 {
+        0 => responder(client).await?,
+        1 => requester(client).await?,
+        _ => unreachable!(),
+    }
+    Ok(())
+}
+
+async fn build_node(store: Store, addr: SocketAddr) -> Result<Node> {
+    // let (server_config, server_cert) = krakensync::network::read_localhost_config()?;
```

Comment:
drop this?

Reply:
ahhh, this whole file should be removed.